f2/src/forms/ContactInfoForm.js

```diff
@@ -28,8 +46,25 @@ export const ContactInfoForm = ({ onSubmit }) => {
     <Form
       initialValues={contactInfo}
       onSubmit={onSubmit}
-      render={({ handleSubmit }) => (
+      validate={validate}
+      render={({
+        handleSubmit,
+        values,
+        errors,
+        submitFailed,
+        hasValidationErrors,
+      }) => (
         <Stack as="form" onSubmit={handleSubmit} shouldWrapChildren spacing={6}>
+          {submitFailed && hasValidationErrors ? (
+            <ErrorSummary onSubmit={handleSubmit} errors={errors} />
+          ) : null}
+          <FormArrayControl
+            name="consentOptions"
+            errorMessage={
+              ((<Trans id="contactinfoForm.email.warning" />),
+              (<Trans id="contactinfoForm.phone.warning" />))
+            }
+          ></FormArrayControl>
```

Comment:
We don't need this here. you can move the Trans tags up into a mark lingui ids block like we have on other pages! :
```jsx
{false ? ( // mark ids for lingui
  <div>
    <Trans id="howDidTheyReachYouLabel.question1" />
  </div>
) : null}
```